pkgs/development/python-modules/aioimaplib/default.nix

```diff
@@ -37,17 +35,24 @@ buildPythonPackage rec {
     pytz
   ];
 
-  disabledTests = [
-    # TimeoutError
-    "test_idle_start__exits_queue_get_without_timeout_error"
-  ];
+  disabledTests =
+    [
+      # TimeoutError
+      "test_idle"
+      "test_idle_start__exits_queue_get_without_timeout_error"
+    ]
+    ++ lib.optionals stdenv.isDarwin [
```

Comment:
```suggestion
    ++ lib.optionals stdenv.hostPlatform.isDarwin [
```

Reply:
I've been using the long form, and had the short form just suggested by a reviewer. I'll keep using the long form from here.

Fixed.

Reply:
Please always use the long form: https://github.com/NixOS/nixpkgs/pull/341407